.gitignore

```diff
@@ -30,3 +31,6 @@ ehthumbs.db
 #############
 .htaccess
 
+# Composer #
+#############
+vendor/
```

Comment:
> When merging this, the Wiki should also be updated to include PHPCompatibility.

I've updated it. Since I only added two extra files to the .gitignore file, I'm going to ahead and send this off to @michaeltorbert for final review.